rct229/rules/section21/section21rule9.py

```diff
@@ -0,0 +1,83 @@
+from rct229.rule_engine.rule_base import RuleDefinitionBase
+from rct229.rule_engine.rule_list_indexed_base import RuleDefinitionListIndexedBase
+from rct229.rule_engine.user_baseline_proposed_vals import UserBaselineProposedVals
+from rct229.utils.assertions import getattr_
+from rct229.schema.config import ureg
+from rct229.utils.jsonpath_utils import find_all
+
+APPLICABLE_SYS_TYPES = [
+    "SYS-1",
+    "SYS-5",
+    "SYS-7",
+    "SYS-11.2",
+    "SYS-12",
+    "SYS-1A",
+    "SYS-7A",
+    "SYS-11.2A",
+    "SYS-12A",
+]
+PUMP_POWER_PER_FLOW_RATE = 19.0 * ureg("W/gpm")
```

Comment:
Would it make sense to call this `REQUIRED_PUMP_POWER_PER_FLOW_RATE`?  I think that would help the readability of the rule_check.